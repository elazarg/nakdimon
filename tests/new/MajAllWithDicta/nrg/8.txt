כְּאִלּוּ בְּדִיּוּק הֵן שָׁלוֹשׁ הַמָּנוֹת שֶׁאוֹתָן בָּחַרְתִּי עֲבוּרְכֶם הַשָּׁבוּעַ. בְּמִתְכַּוֵּן הָרִאשׁוֹן נִשְׁתַּמַּשׁ בְּנֵתַח אוסובוקו, בִּשְׁנֵי נִשְׁתַּמַּשׁ בגלילת בָּשָׂר אוֹ בְּנֵתַח בֹּקֶר צָלוּי, ובמתכון הַשְּׁלִישִׁי נִשְׁתַּמַּשׁ בָּעוֹף שָׁלֵם. כָּל הַשְּׁלוֹשָׁה מוּנָּחִים בְּתַבְנִית תַּנּוּר אוֹ בִּכְלִי חָסִין אֵשׁ לְצַד שְׁלַל יְרָקוֹת וְתוֹסָפוֹת וְנִשְׁלָחִים לאפייה מְמוּשֶּׁכֶת שֶׁל שָׁלוֹשׁ וָחֵצִי עַד אַרְבַּע שָׁעוֹת. אַחֲרֵי שָׁעָה וָחֵצִי בָּדְקוּ אֶת כְּמוֹת הַנּוֹזְלִים שֶׁל הָרוֹטֶב בְּתַבְנִית, וּמִדֵּי פַּעַם הִשְׁקוּ אוֹ הברישו אֶת הַנְּתָחִים בְּרוֹטֶב.