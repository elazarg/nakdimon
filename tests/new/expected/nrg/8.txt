כָּאֵלּוּ בְּדִיּוּק הֵן שְׁלוֹשׁ הַמָּנוֹת שֶׁאוֹתָן בָּחַרְתִּי עֲבוּרְכֶם הַשָּׁבוּעַ.
בַּמַּתְכּוֹן הָרִאשׁוֹן נִשְׁתַּמֵּשׁ בְּנֵתַח אוֹסוֹבּוּקוֹ, בַּשֵּׁנִי נִשְׁתַּמֵּשׁ בִּגְלִילַת בָּשָׂר אוֹ בְּנֵתַח בָּקָר צָלוּי, וּבַמַּתְכּוֹן הַשְּׁלִישִׁי נִשְׁתַּמֵּשׁ בְּעוֹף שָׁלֵם.
כָּל הַשְּׁלוֹשָׁה מוּנָּחִים בְּתַבְנִית תַּנּוּר אוֹ בִּכְלִי חֲסִין אֵשׁ לְצַד שְׁלַל יְרָקוֹת וְתוֹסָפוֹת וְנִשְׁלָחִים לַאֲפִיָּיה מְמוּשֶּׁכֶת שֶׁל שָׁלוֹשׁ וָחֵצִי עַד אַרְבַּע שָׁעוֹת.
אַחֲרֵי שָׁעָה וָחֵצִי בִּדְקוּ אֶת כַּמּוּת הַנּוֹזְלִים שֶׁל הָרוֹטֶב בַּתַּבְנִית, וּמִדֵּי פַּעַם הַשְׁקוּ אוֹ הַבְרִישׁוּ אֶת הַנְּתָחִים בְּרוֹטֶב.